website/docs/guides/getting_started.html.markdown

```diff
@@ -156,6 +156,56 @@ resource "newrelic_alert_policy_channel" "alert_policy_email" {
 
 This example will send an email to the specified recipients whenever the associated alert condition is triggered. If you would like to send notifications via different modalities, such as Slack, you can configure updating the `type` in your [alert channel](https://www.terraform.io/docs/providers/newrelic/r/alert_channel.html).
 
+## A Note About Secrets
+
+As part of a `newrelic` resource, there is often some amount of configuration
+that is required in order for a resource to reach its full potential.  In some
+cases, once a given entity is created, API results will obscure the values for
+items that are deemed to be secret.  As a result, Terraform is unable to make
+an accurate detection of a resource state, and so marks a resource as changed
+for every run.
+
+Consider the following example.
+
+```hcl
+resource "newrelic_alert_channel" "slack" {
+  name = "slack"
+  type = "slack"
+
+  configuration = {
```

Comment:
The `configuration` block has been deprecated. We want to encourage others to use the new `config` block instead. Could we update these docs to reflect that too potentially?

Reply:
Indeed yes.  I've updated the doc to use `config` in place of `configuration` and checked for other references.